trigger/project_triggers.go

```diff
@@ -38,10 +38,13 @@ func TriggerDownstreamVersion(args ProcessorArgs) (*model.Version, error) {
 	var pp *model.ParserProject
 	var pc *model.ProjectConfig
 	if args.ConfigFile != "" {
-		proj, pp, pc, err = makeDownstreamProjectFromFile(args.DownstreamProject, args.ConfigFile)
+		projectInfo, err := makeDownstreamProjectFromFile(args.DownstreamProject, args.ConfigFile)
 		if err != nil {
 			return nil, errors.WithStack(err)
 		}
+		proj = projectInfo.Project
```

Comment:
We use these variables to create a new ProjectInfo object at line 58 so this feels redundant; instead of initializing each of these individually, we can just create a projectInfo at the top and then only assign Ref at 58